frontend/src/model/Meeple.ts

```diff
@@ -0,0 +1,17 @@
+import Locations from '../constants/locations';
+class Player {}
+class Tile {}
+
+class Meeple {
+  playerRef: Player;
```

Comment:
let's change this property name to "player" instead of playerRef. It would be much cleaner naming

Please do this for tileRef as well